README.md

```diff
@@ -23,6 +25,23 @@ func main() {
 }
 ```
 
+#### With exasol config
+
+```go
+package main
+
+import (
+	"database/sql"
+	
+	"github.com/exasol/exasol-driver-go"
+)
+
+func main() {
+	db, err := sql.Open("exasol", exasol.NewConfig("<username>", "<password>").Port(<port>).Host("<host>").String())
```

Comment:
The example would look better if we are consistent in naming. Either we use `exasol` as a variable name everywhere or we pick another name and change it in all other examples. I would suggest avoiding unnecessary abbreviations, but `database` also sounds fine to me

Reply:
I will change everything to `database`, `exasol` is not possible, because of the import